editor/code_editor.cpp

```diff
@@ -1358,14 +1358,14 @@ void CodeTextEditor::move_lines_up() {
 			int cursor_column = text_editor->get_caret_column(c);
 
 			if (!text_editor->has_selection(c)) {
-				caret_group_parameters.push_back(Vector<int>{ -1, -1, -1, -1, cursor_line, cursor_column });
-				continue;
+				caret_group_parameters.push_back(Vector<int>{ -1, -1, -1, -1, -1, -1, cursor_line, cursor_column });
+			} else {
```

Comment:
Oops! Sure thing.

Reply:
Wait a sec, no, that shouldn't be necessary because I changed it to an if-else, right? If this code runs, the `else` is then skipped and it goes to the next loop iteration anyway. This is how `move_lines_down()` does it.